src/common/bilateral.c

```diff
@@ -449,6 +448,7 @@ void dt_bilateral_slice_to_output(const dt_bilateral_t *const b, const float *co
       out[index] = MAX(0.0f, out[index] + Lout);
     }
   }
+  fprintf(stderr,"bilat_slice_to %.3f ms\n",1000.0*(dt_get_wtime()-start));
```

Comment:
All the `fprintf(stderr` have recently been changed, so should be `dt_print(DT_DEBUG_ALWAYS,`.

Reply:
Let me do the change. I have already written the RELEASE_NOTES entries...

Reply:
Even easier, this is debug code. I'll remove.

Reply:
Oops,  sorry about that -- yeah, that was my timing code to check whether I was actually improving the speed.